historydag/utils.py

```diff
@@ -682,6 +682,58 @@ def edge_func(n1, n2):
     return kwargs
 
 
+def one_sided_rfdistance_funcs(reference_dag: "HistoryDag"):
```

Comment:
I think this looks great! I think the argument `reference_dag` should be called `reference_history`, since even though it's a history DAG object, it's expected to be tree-shaped (hence an history).

Also, we should make it very clear in the docstring whether the reference_history is expected to be multifurcating or not, when we're trying to detect resolutions of multifurcating trees.